{"uid":"de5a104ff562728a","name":"test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","fullName":"tests.catalog.options.test_HU11_ES_POST_ProductOption#test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","historyId":"09c4f63e9a9984c5936c41cb895fd765","time":{"start":1755555888245,"stop":1755555888489,"duration":244},"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU1NTU3OTgsImV4cCI6MTc1NTU1OTM5OCwicm9sZX...yd0ROj_20YSoFasfkDP65I7hy9AX-i4vzzQhWD2JQkMuXpHpLpRzc4UbS8WaUHzfX3BVEHUgolhcJOw3d5DsfKroQxS1xbhzWrVFL93NbqvXjIx8'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.medium\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:184: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755555798344,"stop":1755555798345,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"auth_headers","time":{"start":1755555798345,"stop":1755555798989,"duration":644},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_options","time":{"start":1755555888245,"stop":1755555888245,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTU1NTU3OTgsImV4cCI6MTc1NTU1OTM5OCwicm9sZX...yd0ROj_20YSoFasfkDP65I7hy9AX-i4vzzQhWD2JQkMuXpHpLpRzc4UbS8WaUHzfX3BVEHUgolhcJOw3d5DsfKroQxS1xbhzWrVFL93NbqvXjIx8'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.medium\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:184: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","steps":[],"attachments":[{"uid":"9dd78a2b45e513c","name":"log","source":"9dd78a2b45e513c.txt","type":"text/plain","size":2183}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"setup_add_options::0","time":{"start":1755555888495,"stop":1755555888495,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"medium"},{"name":"tag","value":"boundary"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.catalog.options"},{"name":"suite","value":"test_HU11_ES_POST_ProductOption"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2424-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.catalog.options.test_HU11_ES_POST_ProductOption"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":11,"passed":0,"unknown":0,"total":11},"items":[{"uid":"8a42818c09767986","reportUrl":"https://Elv500.github.io/Softsign/97//#testresult/8a42818c09767986","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755555658813,"stop":1755555658977,"duration":164}},{"uid":"e7b5aae4effa7dc5","reportUrl":"https://Elv500.github.io/Softsign/96//#testresult/e7b5aae4effa7dc5","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755555043454,"stop":1755555043696,"duration":242}},{"uid":"ee44c9425be007b7","reportUrl":"https://Elv500.github.io/Softsign/95//#testresult/ee44c9425be007b7","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755553468957,"stop":1755553469202,"duration":245}},{"uid":"c98628a162341333","reportUrl":"https://Elv500.github.io/Softsign/94//#testresult/c98628a162341333","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755553162405,"stop":1755553162657,"duration":252}},{"uid":"14e1463b79688404","reportUrl":"https://Elv500.github.io/Softsign/93//#testresult/14e1463b79688404","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755552771067,"stop":1755552771428,"duration":361}},{"uid":"f804f30e3aa211c1","reportUrl":"https://Elv500.github.io/Softsign/92//#testresult/f804f30e3aa211c1","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755552466303,"stop":1755552466793,"duration":490}},{"uid":"2092aacdbe421d05","reportUrl":"https://Elv500.github.io/Softsign/77//#testresult/2092aacdbe421d05","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755547107532,"stop":1755547107909,"duration":377}},{"uid":"8a5a0875afbb25d4","reportUrl":"https://Elv500.github.io/Softsign/74//#testresult/8a5a0875afbb25d4","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755546992466,"stop":1755546993193,"duration":727}},{"uid":"3efd1e3643b2a76c","reportUrl":"https://Elv500.github.io/Softsign/70//#testresult/3efd1e3643b2a76c","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755546839510,"stop":1755546839874,"duration":364}},{"uid":"8db98430c6c6fa81","reportUrl":"https://Elv500.github.io/Softsign/57//#testresult/8db98430c6c6fa81","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755544227625,"stop":1755544228436,"duration":811}}]},"tags":["boundary","functional","medium"]},"source":"de5a104ff562728a.json","parameterValues":[]}